release-notes-1.1.1.md

```diff
@@ -1,26 +1,62 @@
-# Overview to Ballerina 1.1.1
+# Overview of jBallerina 1.1.1
```

Comment:
IMO, we need to be consistent on using `jBallerina` or `JBallerina`. Previous release note has used 'JBallerina'. https://ballerina.io/downloads/release-notes/ . I think `jBallerina` is the correct way and @hasithaa @Kishanthan wdyt?

Reply:
Yeah, I think jBallerina is better.

Reply:
+1 for jBallerina

Reply:
It has to be JBallerina because when you type ballerina -v we show version as jBallerina from last release